### PR TITLE
[bn] Adding  content/bn/docs/contribute/_index.md  for Bengali Localization

### PR DESCRIPTION
**Pull request by Saber382**
Signed-off-by: Saber Imtiaz saberimtiazmitul@gmail.com
Localizes ``content/bn/docs/contribute/_index.md``  to Bengali
This is an intial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

File: content/bn/docs/contribute/_index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+কুবারনেটস সম্প্রদায়ে কার্যকরভাবে কাজ করার জন্য।
+
+
+ডকুমেন্টেশনের সাথে জড়িত হতে:
+
+1. CNCF [কন্ট্রিবিউটর লাইসেন্স চুক্তি](https://github.com/kubernetes/community/blob/master/CLA.md) স্বাক্ষর করুন।
+1. [ডকুমেন্টেশন রিপোজিটরি](https://github.com/kubernetes/website) এবং ওয়েবসাইটের [স্ট্যাটিক সাইট জেনারেটর](https://gohugo.io) এর সাথে নিজেকে পরিচিত করুন । 
+1. নিশ্চিত করুন যে আপনি প্রাথমিক প্রক্রিয়াগুলি বুঝতে পেরেছেন৷
+   [একটি পুল অনুরোধ খোলার](/docs/contribute/new-content/open-a-pr/) এবং
+   [পরিবর্তন পর্যালোচনা করার](/docs/contribute/review/reviewing-prs/)।
```
Comment (anubha-v-ardhan) [inline content/bn/docs/contribute/_index.md:50]:
```suggestion
1. CNCF [কন্ট্রিবিউটর লাইসেন্স চুক্তি](https://github.com/kubernetes/community/blob/master/CLA.md) স্বাক্ষর করুন।
2. [ডকুমেন্টেশন রিপোজিটরি](https://github.com/kubernetes/website) এবং ওয়েবসাইটের [স্ট্যাটিক সাইট জেনারেটর](https://gohugo.io) এর সাথে নিজেকে পরিচিত করুন । 
3. নিশ্চিত করুন যে আপনি প্রাথমিক প্রক্রিয়াগুলি বুঝতে পেরেছেন৷
   [একটি পুল অনুরোধ খোলার](/docs/contribute/new-content/open-a-pr/) এবং
   [পরিবর্তন পর্যালোচনা করার](/docs/contribute/review/reviewing-prs/)।
```

File: content/bn/docs/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
```
Comment (anubha-v-ardhan) [inline content/bn/docs/contribute/_index.md:51]: There should be a mermaid diagram here (refer [English version](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/_index.md?plain=1#L55)). We should avoid omitting anything as much as we can

File: content/bn/docs/contribute/_index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Kubernetes ডকুমেন্টেশন অবদানকারী:
+
+- বিদ্যমান সামগ্রী উন্নত করুন
+- নতুন কন্টেন্ট তৈরি করুন
+- ডকুমেন্টেশন অনুবাদ করুন
+- Kubernetes রিলিজ চক্রের ডকুমেন্টেশন অংশগুলি পরিচালনা এবং প্রকাশ করুন
+
+<!-- body -->
+
+## শুরু করুন (Get Started)
```
[the remark this answers is not in the document]
Comment (Saber382) [reply content/bn/docs/contribute/_index.md:34]: No. Just to make it easier for others to know what it actually means in the local language. Need to modify?

[the remark this answers is not in the document]
Comment (anubha-v-ardhan) [reply content/bn/docs/contribute/_index.md:34]: I don't read Bengali, I'm not in a position to say if it's necessary or not. But if we're adding English headings in brackets, there should be a standard maintained thoughout the Bengali localization.

I would recommend taking this on slack (#kubernetes-docs-bn) and discussing with folks who read and write Bengali.

[the remark this answers is not in the document]
Comment (Saber382) [reply content/bn/docs/contribute/_index.md:34]: Okay sure!

File: content/bn/docs/contribute/_index.md (hunk 2 above)
Comment (Saber382) [reply content/bn/docs/contribute/_index.md]: @anubha-v-ardhan Updated all of the contents

**Comment by Saber382** [comment]
@anubha-v-ardhan  can you check the whole file again? I have tried to update as you suggested.

File: content/bn/docs/contribute/_index.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
+---
+content_type: ধারণা
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:2]: The value here shouldn't be localized.

File: content/bn/docs/contribute/_index.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+{{</ mermaid >}}
+***চিত্র - আপনার প্রথম অবদানের জন্য প্রস্তুতি***
+
+
+- [কন্ট্রিবিউশন ওভারভিউ](/docs/contribute/new-content/overview/) পড়ুন এবং
+  আপনি  যেসব উপায়ে অবদান রাখতে পারেন সেসব সম্পর্কে জানুন । 
+- [`kubernetes/website` সমস্যা তালিকা](https://github.com/kubernetes/website/issues/) চেক করুন যা
+  ভালো এন্ট্রি পয়েন্ট তৈরি করে এমন সমস্যার জন্য।
+- [GitHub ব্যবহার করে একটি পুল অনুরোধ খুলুন](/docs/contribute/new-content/open-a-pr/#changes-using-github)
+  বিদ্যমান ডকুমেন্টেশনে এবং GitHub-এ ফাইল করা সমস্যা ( filing issues in GitHub) সম্পর্কে আরও জানুন।
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:150]:
```suggestion
বিদ্যমান ডকুমেন্টেশনে এবং GitHub-এ ফাইল করা সমস্যা (filing issues in GitHub) সম্পর্কে আরও জানুন।
```
?

File: content/bn/docs/contribute/_index.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+- অন্য Kubernetes সম্প্রদায়ের সদস্যসদের  থেকে
+  সঠিকতা এবং ভাষার জন্য [পুলের অনুরোধ পর্যালোচনা করুন](/docs/contribute/review/reviewing-prs/)  
+- Kubernetes [content](/docs/contribute/style/content-guide/)  এবং [শৈলী নির্দেশিকা](/docs/contribute/style/style-guide/) পড়ুন
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:153]: I'd expect to see “content” localized. For example, does  “ওয়েব সামগ্রী” make sense?  (that's a machine translation&mdash;I don't speak Bangla).

File: content/bn/docs/contribute/_index.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+subgraph second[পুনঃমূল্যায়ন]
+direction TB
+   T[ ] -.-
+   D[দেখুন<br>K8s/website<br>ভান্ডার] --- E[দেখুন<br>Hugo static সাইট<br>generator]
+   E --- F[মৌলিক <br>GitHub কমান্ড বুঝুন]
+   F --- G[পর্যালোচনা করো খোলা পুল রিকোয়েস্ট<br>এবং পর্যালোচনা পরিবর্তন করুন <br>প্রসেস]
+end
+
+subgraph first[Sign up]
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:75]: I'd expect to see “Sign up” localized.

File: content/bn/docs/contribute/_index.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+- ভান্ডার থেকে শিখুন  [একটি স্থানীয় ক্লোন (local clone) থেকে কাজ করা](/docs/contribute/new-content/open-a-pr/#fork-the-repo) ।
+-  ডকুমেনটেশন কর [একটি রিলিজের বৈশিষ্ট্য](/docs/contribute/new-content/new-features/)।
+- [SIG ডক্স](/docs/contribute/participate/) এ অংশগ্রহণ করুন এবং একজন 
+  [সদস্য বা পর্যালোচক](/docs/contribute/participate/roles-and-responsibilities/) হন।
+                       
+- একটি [স্থানীয়করণ](/docs/contribute/localization/) দিয়ে শুরু করুন বা সাহায্য করুন।
+
+## SIG ডক্সের সাথে জড়িত হন 
+
+[SIG ডক্স](/docs/contribute/participate/) হল অবদানকারীদের গ্রুপ যারা
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:169]:
```suggestion
[SIG Docs](/docs/contribute/participate/) হল অবদানকারীদের গ্রুপ যারা
```

File: content/bn/docs/contribute/_index.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+- Kubernetes [content](/docs/contribute/style/content-guide/)  এবং [শৈলী নির্দেশিকা](/docs/contribute/style/style-guide/) পড়ুন
+   যাতে আপনি অবহিত মন্তব্য করতে পারেন।
+- [পৃষ্ঠা বিষয়বস্তুর প্রকার](/docs/contribute/style/page-content-types/) সম্পর্কে জানুন
+  এবং [Hugo শর্ট কোডস](/docs/contribute/style/hugo-shortcodes/)।
+
+## পরবর্তী পদক্ষেপ 
+
+- ভান্ডার থেকে শিখুন  [একটি স্থানীয় ক্লোন (local clone) থেকে কাজ করা](/docs/contribute/new-content/open-a-pr/#fork-the-repo) ।
+-  ডকুমেনটেশন কর [একটি রিলিজের বৈশিষ্ট্য](/docs/contribute/new-content/new-features/)।
+- [SIG ডক্স](/docs/contribute/participate/) এ অংশগ্রহণ করুন এবং একজন
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:162]:
```suggestion
- [SIG Docs](/docs/contribute/participate/) এ অংশগ্রহণ করুন এবং একজন 
```
It's OK to add an aside to explain what “Docs” (or indeed “SIG”) means. However, the group is called SIG Docs and not SIG ডক্স.

File: content/bn/docs/contribute/_index.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+<!-- overview -->
+
+*Kubernetes নতুন এবং অভিজ্ঞ সকল অবদানকারীদের থেকে উন্নতিকে (immprovements) স্বাগত জানায়!*
+
+{{< note >}}
+সাধারণভাবে Kubernetes অবদান সম্পর্কে আরও জানতে, দেখুন
+[অবদানকারীদের জন্য ডকুমেন্টেশন](https://www.kubernetes.dev/docs/)।
+{{< /note >}}
+
+এই ওয়েবসাইটটি [Kubernetes SIG ডক্স](/docs/contribute/#get-involved-with-sig-docs) দ্বারা রক্ষণাবেক্ষণ করা হয় ।
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:23]:
```suggestion
এই ওয়েবসাইটটি [Kubernetes SIG Docs](/docs/contribute/#get-involved-with-sig-docs) দ্বারা রক্ষণাবেক্ষণ করা হয় ।
```

File: content/bn/docs/contribute/_index.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+- নতুন কন্টেন্ট তৈরি করুন
+- ডকুমেন্টেশন অনুবাদ করুন
+- Kubernetes রিলিজ চক্রের ডকুমেন্টেশন অংশগুলি পরিচালনা এবং প্রকাশ করুন
+
+<!-- body -->
+
+## শুরু করুন 
+
+যে কেউ ডকুমেন্টেশন সম্পর্কে একটি সমস্যা (issue)  খুলতে পারে, বা   পরিবর্তন করতে পারে
+ পুল রিকুয়েস্ট(Pull Request) দেয়ার মাধ্যমে
```
Comment (sftim) [inline content/bn/docs/contribute/_index.md:37]:
```suggestion
পুল রিকুয়েস্ট(pull request) দেয়ার মাধ্যমে
```

**Review by sftim — COMMENTED** [review]
Some feedback

**Review by anubha-v-ardhan — COMMENTED** [review]
/language bn

**Comment by sftim** [comment]
@Saber382 you now can (and should) update this PR to target the branch [_dev-1.23-bn.1_](https://github.com/kubernetes/website/tree/dev-1.23-bn.1). Would you have time to do that?

**Review by Imtiaz1234 — APPROVED** [review]
/lgtm

**Pull request merged**